comp-mng-client/src/app/constants/constants.ts

```diff
@@ -0,0 +1,9 @@
+export const URL = {
+  API_AUTH: 'http://localhost:8080/api/auth/login',
+  API_REGISTER: 'http://localhost:8080/api/auth/register'
```

Comment:
Voor extra bonuspunten kan je de host (http://localhost:8080) configurabel maken in een property file

Reply:
Hier ga ik zeker even naar kijken, heb je wellicht leesvoer voor me om dit voor elkaar te krijgen?